src/containers/backpack.jsx

```diff
@@ -107,7 +107,10 @@ class Backpack extends React.Component {
                     username: this.props.username,
                     ...payload
                 }))
-                .then(this.refreshContents);
+                .then(this.refreshContents)
```

Comment:
`refreshContents` doesn't appear to return a Promise, so I'm confused how `catch()` can catch the rejection.

Reply:
Oh this error in particular is thrown in the payloader, not refreshContents. This stuff changed with the pagination PR though, so I'm going to rebase and push. None of the error stuff will be in this PR